### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by abdullahcalisir12**
https://github.com/whynotearth/foodouken.com/issues/196
https://github.com/whynotearth/foodouken.com/issues/171
https://github.com/whynotearth/foodouken.com/issues/179

### CODE REVIEW

**Comment by sally** [comment]
this is currently labeled as WIP, should I still review it? @abdullahcalisir12

**Comment by abdullahcalisir12** [comment]
> this is currently labeled as WIP but I was tagged for review by Chris, should I still review it? @abdullahcalisir12

yes please but only code standarts, not view and functionality.

**Comment by sally** [comment]
@abdullahcalisir12 Okay, in that case I'll wait until it's no longer in 'WIP' status. Also it looks like it's waiting on #199 to be merged to be more of a focused review, which gives me more reason to wait. :)

**Comment by abdullahcalisir12** [comment]
> @abdullahcalisir12 Okay, in that case I'll wait until it's no longer in 'WIP' status. Also it looks like it's waiting on #199 to be merged to be more of a focused review, which gives me more reason to wait. :)

https://github.com/whynotearth/foodouken.com/pull/199 this PR merged
for dashboard still needs time I think today or tomorrow will be ready to review

**Comment by abdullahcalisir12** [comment]
@sally you can review now!

File: postcss.config.js [hunk 1]
```diff
@@ -20,10 +17,13 @@ const purgecss = postcssPurgecss({
...
+};
+
 module.exports = {
-  plugins: [
-    tailwindcss,
-    autoprefixer,
-    ...(process.env.NODE_ENV === 'production' ? [purgecss] : [])
-  ]
+  plugins: [tailwindcss, postcssNested, pxtorem(pxtoremOptions), autoprefixer]
+  // , ...(process.env.NODE_ENV === 'production' ? [purgecss] : [])
```
Comment (atharva3010) [inline postcss.config.js:28]: This disables purgeCSS in production, please enable it.

File: postcss.config.js [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
-    const contentWithoutStyleBlocks = content.replace(
-      /<style[^]+?<\/style>/gi,
-      ''
-    );
-    return (
-      contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/]+/g) || []
-    );
+    const contentWithoutStyleBlocks = content.replace(/<style[^]+?<\/style>/gi, '');
+    return contentWithoutStyleBlocks.match(/[A-Za-z0-9-_/:]*[A-Za-z0-9-_/]+/g) || [];
   },
```
Comment (atharva3010) [inline postcss.config.js:12]: have we changed the lint rules?

**Comment by atharva3010** [comment]
A lot of utilities in the styles are from Volkswagen

**Comment by atharva3010** [comment]
I think using a carousel might be an overkill for this, we can make that component ourselves easily with about the same effort as it took for us to integrate the carousel.

**Comment by atharva3010** [comment]
Px to rem too, it's not as big of an issue to just write values in rem ourselves. Browtricks's tailwind config is a good example. It's a good idea to not rely on abstractions a lot and try to do most of the things that we can, ourselves.

**Comment by atharva3010** [comment]
In my opinion, we should adjust the values of default font sizes in the main config file, so that our whole app gets synced with the designs.

**Comment by abdullahcalisir12** [comment]
This PR is closed and replaced with #230

**Pull request closed without merging**